Core/Base/ConsumerReactor.py

```diff
@@ -0,0 +1,62 @@
+"""
+  DIRAC class to execute consumers
+
+  All DIRAC consumers must inherit from the basic class ConsumerModule
+
+  The class is created based on the AgentReactor example.
+  In the most common case, DIRAC consumers are executed using the dirac-consumer command.
+  dirac-consumer accepts a list positional arguments. These arguments have the form:
+  [DIRAC System Name]/[DIRAC Consumer Name]
+  dirac-consumer then:
+  - produces a instance of ConsumerReactor
+  - loads the required module using the loadModule method
+  - starts the consumer itself using the ConsumerReactor.go method
+
+"""
+
+from DIRAC.Core.Base.ConsumerModule import ConsumerModule
+from DIRAC.Core.Base.private.ModuleLoader import ModuleLoader
+from DIRAC.WorkloadManagementSystem.Consumer.ConsumerTools import getConsumerSection
+from DIRAC import S_OK
+from DIRAC.Core.Utilities.DErrno import DError
+import errno
+
+def loadConsumerModule( consumerModuleName, hideExceptions = False ):
+  """Loads the consumer module.
+  Args:
+    consumerModuleName(str):
+  """
+  loader = ModuleLoader( "Consumer", getConsumerSection, ConsumerModule )
+  #The function loadModules takes as the first argument, the list
+  #of modules to load. Even that we actually have only one consumer
+  #module we must transform it to a list.
+  moduleList = [ consumerModuleName ]
```

Comment:
Maybe not needed to define a new variable just for that, put it directly in the call no ?